hack/golangci-lint.sh

```diff
@@ -1,9 +1,8 @@
 #!/bin/sh
 set -e
 
-GOLANGCI_VERSION="1.43.0"
-
 OUTPUT_PATH=${1:-./bin/golangci-lint}
+GOLANGCI_VERSION=${2:-1.43.0}
```

Comment:
Actually we don't want to do this, because we verify the SHA256 of the downloaded tar files and they will fail for any other version. We should drop the version in the `Makefile`.

Reply:
Removed